### DISCUSSION TITLE
Is there a way to configure vs code to line wrap tailwind?

### DISCUSSION THREAD

**Discussion by mrhoodz** [tailwindlabs/tailwindcss#10800] [Help] [answered]
All l've seen is a fold extension which is useful to hide the long utilizes class string but not so useful during the styling phase using the utilizes. I find myself scrolling left and right for small tweaks and  changes all the time which really slows down the process.

**Comment by wongjn** [comment, marked as answer]
There is a "general" editor setting to word wrap in VSCode, `editor.wordWrap`.

**Reply by lgkonline** [reply]
Thank you! This should maybe mentioned on https://tailwindcss.com/docs/editor-setup.
